docs/docs/rest-api/public/api/v2/examples/app.json

```diff
@@ -73,6 +73,18 @@
       "timeoutSeconds": 20
     }
   ],
+  "readinessChecks": [
+    {
+      "name": "myReadyCheck",
+      "protocol": "HTTP",
+      "path": "/v1/plan"
+      "portName": "http"
+      "interval": 10,
```

Comment:
hmmm, seeing this we maybe should make the unit clear.

Why are the colons highlighted as red?